config-repo/clusterregistry/clusters.yaml

```diff
@@ -18,3 +18,8 @@ metadata:
   name: anthos-sample-cluster1
   labels:
     environment: asm
+---
```

Comment:
This part should be removed because ASD is currently only deploying with one cluster.

Reply:
ah true, updated.